_includes/tutorials/connect-add-key-to-source/kafka/code/tutorial-steps/dev/consume-topic.sh

```diff
@@ -1,3 +1,3 @@
-docker exec -i kafkacat kafkacat -b broker:9092 -t postgres_cities \
+docker exec -i kcat kcat -b broker:9092 -t postgres_cities \
```

Comment:
I think we removed the superfluous `kcat` after the image name when we renamed the confluentinc/kafkacat->kcat docker image (and it was never there for the edenhill docker images).

So, all docker commands that use the edenhill/kcat image, or the newer confluentinc/kcat image, should remove the extra  `kcat` argument

Reply:
@edenhill the KT usage of `kcat` (and `kafkcat` before) seems to have been overriding the `entrypoint`, so the extra `kcat` is required.

```
  kcat:
    image: edenhill/kcat:1.7.1
    container_name: kcat
    links:
      - broker
    entrypoint:
      - /bin/sh
      - -c
      - |
        apk add jq;
        while [ 1 -eq 1 ];do sleep 60;done
```

So removing the extra `kcat` results in an error:

```
docker-compose exec kcat -b broker:9092 -t postgres_cities \     
            -C -s avro -r http://schema-registry:8081 -e \
            -f 'Key     (%K bytes):\t%k\nPayload (%S bytes):\t%s\n--\n'

OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "-b": executable file not found in $PATH: unknown
```